helpers.lua

```diff
@@ -471,6 +484,8 @@ function get_data_for(view)
         or view == "edit.html"
         or view == "password.html"
         or view == "ynhpanel.json" then
```

Comment:
I don't know. this is useless. Required informations are uid, givenName, sn, cn and mail.
maildrop and mailaliases are not needed.